var/spack/repos/builtin/packages/r-rjava/package.py

```diff
@@ -7,16 +7,26 @@
 
 
 class RRjava(RPackage):
-    """Low-level interface to Java VM very much like .C/.Call and friends.
+    """Low-Level R to Java Interface
+
+    Low-level interface to Java VM very much like .C/.Call and friends.
     Allows creation of objects, calling methods and accessing fields."""
 
     homepage = "http://www.rforge.net/rJava/"
     url      = "https://cloud.r-project.org/src/contrib/rJava_0.9-8.tar.gz"
     list_url = "https://cloud.r-project.org/src/contrib/Archive/rJava"
 
+    version('0.9-13', sha256='5b1688f5044476b34f71d868b222ac5fce3a088f0c2b9e4591c1e48f3d8c75f4')
     version('0.9-11', sha256='c28ae131456a98f4d3498aa8f6eac9d4df48727008dacff1aa561fc883972c69')
     version('0.9-8', sha256='dada5e031414da54eb80b9024d51866c20b92d41d68da65789fe0130bc54bd8a')
 
     depends_on('r@2.5:', type=('build', 'run'))
     depends_on('java@1.2:')
```

Comment:
```suggestion
    depends_on('java@2:')
```
Java keeps changing it's version numbering scheme. What was once 1.2 is now 2

Reply:
Ah, I was not aware of that.